cpp/server/handler.cc

```diff
@@ -3,160 +3,77 @@
 #include <algorithm>
 #include <event2/buffer.h>
 #include <event2/http.h>
-#include <event2/keyvalq_struct.h>
 #include <functional>
 #include <gflags/gflags.h>
 #include <glog/logging.h>
-#include <map>
 #include <memory>
+#include <mutex>
+#include <stdint.h>
 #include <stdlib.h>
+#include <string>
 #include <utility>
 #include <vector>
 
-#include "log/cert.h"
-#include "log/cert_checker.h"
-#include "log/cluster_state_controller.h"
 #include "log/frontend.h"
 #include "log/log_lookup.h"
-#include "log/logged_entry.h"
 #include "monitoring/monitoring.h"
 #include "monitoring/latency.h"
-#include "server/json_output.h"
-#include "server/proxy.h"
-#include "util/json_wrapper.h"
-#include "util/thread_pool.h"
-
-namespace libevent = cert_trans::libevent;
-
-using cert_trans::Cert;
-using cert_trans::CertChain;
-using cert_trans::CertChecker;
-using cert_trans::Counter;
-using cert_trans::HttpHandler;
-using cert_trans::JsonOutput;
-using cert_trans::Latency;
-using cert_trans::LoggedEntry;
-using cert_trans::Proxy;
-using cert_trans::ScopedLatency;
+#include "util/libevent_wrapper.h"
+#include "util/sync_task.h"
+#include "util/task.h"
+
+
+DEFINE_int32(max_leaf_entries_per_response, 1000,
+             "maximum number of entries to put in the response of a "
+             "get-entries request");
+DEFINE_int32(staleness_check_delay_secs, 5,
+             "number of seconds between node staleness checks");
+
+
+namespace cert_trans {
```

Comment:
I think if you moved lines 26-33 to line 47, it'd make more sense? Small diff *and* reasonable thing to do? (it might be reasonable now, but you're making my pretty little head hurt!)